cfgov/unprocessed/apps/paying-for-college/js/models/state-model.js

```diff
@@ -105,8 +107,15 @@ const stateModel = {
       stateModel.values[key] = stateModel.textVersions[name][value];
       updateSchoolItems();
     }
+
     // When program values are updated, recalculate, updateView
     if ( name.indexOf( 'program' ) === 0 ) {
+      if ( name === 'programType' && value === 'graduate' ) {
+        stateModel.values.programLevel = 'graduate';
+      } else if ( name === 'programType' ) {
+        stateModel.values.programLevel = 'undergrad';
+      }
+
```

Comment:
It looks like the model value gets updated in this function when programName changes but the update does not get propagated to the UI (the summary view in particular)

Reply:
Whoops, that should be a `stateModel.setValue`!